### DISCUSSION TITLE
Discover

### DISCUSSION THREAD

**Discussion by CFTW24** [community/community#129407] [Enterprise]
### Body

![Screenshot_20240621-230233](https://github.com/community/community/assets/171513148/ab9e2260-e426-4f53-8ae2-5e475eff182b)

**Comment by queenofcorgis** [comment]
Hi @CFTW24 , we're happy you're here! You are more likely to get a useful response if you are explicit about what your question entails, giving a few more details might help someone give you a nudge in the right direction.
